mon-pix/app/components/authentication/sso-selection-form.gjs

```diff
@@ -50,6 +63,15 @@ export default class SsoSelectionForm extends Component {
       <OidcProviderSelector @providers={{this.providers}} @onProviderChange={{this.onProviderChange}} />
 
       {{#if this.hasSelectedProvider}}
+        {{#if this.shouldDisplayAccountRecoveryBanner}}
+          <PixNotificationAlert>
+            {{t
+              "components.authentication.sso-selection-form.should-display-account-recovery-banner.question"
```

Comment:
Comme il n’y a plus qu'une seule traduction, il me semblerait plus plus logique de renommer la clé existante comme ceci :

```suggestion
              "components.authentication.sso-selection-form.should-display-account-recovery-banner"
```

Reply:
Dans le composant GJS et dans les fichiers de traduction, je maintiens qu’il serait bon de faire cette simplification `components.authentication.sso-selection-form.should-display-account-recovery-banner.question`→`components.authentication.sso-selection-form.should-display-account-recovery-banner`.